modules/install/class_import.php

```diff
@@ -47,8 +47,7 @@ public function GetImportHeader($tmp_file_name)
     // See: http://lansuite.orgapage.de/index.php?mod=bugtracker&bugid=1059
     #$this->xml_content = str_replace("''", "'", $this->xml_content);
 
-        ## Get Header-Tag
-        $header = "";
+        // Get Header-Tag
         $this->xml_content_lansuite = $xml->getFirstTagContent("LANsurfer", $this->xml_content, 0);
```

Comment:
I think we can remove the whole LANsurfer code. Project died in 2006 as far as I can see. Not sure anybody would need an import function still. And if they do: they can import with LanSuite 4.2 and then move forward from there

Reply:
Will do. Let me create a ticket for this and handle this in a separate PR.

Reply:
@M4LuZ As a follow up: https://github.com/lansuite/lansuite/issues/222